src/client/Client.ts

```diff
@@ -143,7 +147,10 @@ export class ButtplugClient extends EventEmitter {
       return true;
     }
     case "Error": {
+      // Disconnect and throw an exception with the error message we got back.
+      // This will usually only error out if we have a version mismatch.
```

Comment:
This might be true in the future for servers that don't support downgrades, but right now the client does the version checking.
The most common cases for the server to send an error on connect is if the server already had a connection open.

Reply:
Ah, good point. Added a version check to the ServerInfo return that will disconnect on that case too.